### DISCUSSION TITLE
Can´t change batteries

### DISCUSSION THREAD

**Discussion by bhaal66** [Okabintaro/SubmersedVR#60] [Q&A]
Phenomonal Mod! Just encountered the problem, that I can´t change batteries with my index controllers. When I push the A-Button on the right controller the menu shows all batteries, but unfortunately I can´t change to a different one.

**Comment by davomate63** [comment]
For me is the A button on the left controller, while the Seaglider is selected. But you are right, nothing seems to allow you to traverse the row of batteries that appears. The screen shows xbox-type controls, but I do not have an x-box controller plugged in. None of the index controls seem to affect it

**Comment by davomate63** [comment]
Rather than waiting for an answer, I am going to chuck the flat battery seaglide in storage and build a whole new one. Only requires 1 of lubricant, copper wire & titanium (plus the fresh battery)

**Comment by Okabintaro** [comment]
You might have to change the bindings for the next/prev item on index controllers.

**Reply by davomate63** [reply]
Changed to GG Custom Index Binding for SubmersedVR Mod (thanks GunGryphon) and it works! Left/right on the left thumbstick to move through choices, and right trigger to select. I don't need to throw the flat seaglide away now.

**Reply by sapphireglaceon** [reply]
How do you do that? I've been trying to swap batteries for like a week can this please get fixed.

**Reply by Okabintaro** [reply]
Here is a guide on how to customize the bindings: https://steamcommunity.com/sharedfiles/filedetails/?id=2029205314

**Reply by sapphireglaceon** [reply]
Thank you very much

**Reply by sapphireglaceon** [reply]
I don't get it. How do you set the bindings to swap the batteries?

**Reply by Okabintaro** [reply]
You have to bind the Reload / Swap Battery action to a button on your controller.
https://github.com/Okabintaro/SubmersedVR/blob/master/SubmersedVR/StreamingAssets/SteamVR/actions.json#L271

**Reply by Okabintaro** [reply]
On Oculus controllers that is bound to X by default, see: https://github.com/Okabintaro/SubmersedVR/blob/master/SubmersedVR/StreamingAssets/SteamVR/bindings_oculus_touch.json#L70-L76

**Reply by sapphireglaceon** [reply]
Yeah it's the A button. How do you set the cycle thingy on? That's my problem.

**Reply by Okabintaro** [reply]
Those actions are Move Left/Move Right:

https://github.com/Okabintaro/SubmersedVR/blob/master/SubmersedVR/StreamingAssets/SteamVR/actions.json#L274-L275

**Reply by sapphireglaceon** [reply]
Ohhh I'll try that.

**Comment by addyjohns** [comment]
Same issue, everything works except the D-Pad bindings. Can't do anything that requires D-Pad bindings, like changing batteries. No idea how to rebind or fix this. Oh well.
